host/tests/integration/components/catalog-entry-editor-test.gts

```diff
@@ -55,27 +55,19 @@ module('Integration | catalog-entry-editor', function (hooks) {
     Loader.addRealmFetchOverride(realm);
     await realm.ready;
 
-    await realm.write('person.gts', `
+    await realm.write('pet.gts', `
```

Comment:
I'd like to see an additional test that explorers a missing composite field value. perhaps the `Pet` card can have an `owner` field that is a `Person` card. The test would be to create a catalog entry for this new `Pet` card, but not fill out the `owner` field. Ideally we should see that the value for `owner` in the underlying file is not an empty object. `undefined`, while a valid value in a POJO, is not valid in a JSON file. I believe the way our serializer works is that if a field value is `undefined` then it will just omit the field in the resulting .json file.